content/docs/connectors/oauth.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Authentication Through an Generic OAuth 2.0 Provider"
+linkTitle: "Generic OAuth 2.0"
+description: ""
+date: 2021-03-15
+draft: false
```

Comment:
I think we should make it a draft and merge it to master. Once the feature is released, we can flip the switch.

Reply:
This would also be a great opportunity to figure #92 out. @nate-double-u do you think you can help us out with that?

Reply:
I'd be happy to take a look

Reply:
> I think we should make it a draft and merge it to master. Once the feature is released, we can flip the switch.

Have to be careful with that -- this PR touches a couple files